src/main/scala/it/pagopa/interop/apigateway/api/impl/package.scala

```diff
@@ -203,13 +205,6 @@ package object impl extends SprayJsonSupport with DefaultJsonProtocol {
       allAttributes.flatMap(attribute => attribute.map(_.id)).toSet
     }
 
-    def isVerified(attribute: AgreementManagementApiVerifiedAttribute): Boolean = attribute.verified.contains(true)
-
-    def isInTimeRange(attribute: AgreementManagementApiVerifiedAttribute): Boolean =
-      attribute.verificationDate.zip(attribute.validityTimespan).fold(true) { case (verDate, offset) =>
-        OffsetDateTime.now().isBefore(verDate.plusSeconds(offset))
-      }
-
     def attributeUUIDSummary(
```

Comment:
the logic tied to this function is no more valid.
I'm fixing it in a separated PR

Reply:
ok